WTWSMS/history/characters/hunnic.txt

```diff
@@ -235,10 +235,10 @@
 	religion="tengri_pagan"
 	culture="hunnic"
 	trait="strong"
-	trait="wroth"	
+	trait="wroth"
 	trait="brilliant_strategist"
 	father = 76
-	320.1.2 = {
+	336.1.2 = {
 		birth="320.1.2"
```

Comment:
This doesn't work, 336 and 320 don't correspond.

Reply:
My bad, will fix.

Reply:
Done.

Reply:
As I said, it should be 320, it is rather the date of birth of his father that should be changed.

Reply:
Alright, didn't know you edited the first comment.

Reply:
Done.

Reply:
No problems.